.drone.star

```diff
@@ -438,3 +440,10 @@ def publish():
             ]
         }
     }
+
+# replaces reserved and unsafe url characters with '-'
+# reserved: & $ + , / : ; = ? @ #
+# unsafe: <space> < > [ ] { } | \ ^ %
+def sanitizeBranchName(BRANCH_NAME=''):
+    REGEX='[][&$+,/:;=?@#[:space:]<>{}|^%\\\]'
```

Comment:
Love this regex :) Somebody fell asleep end their head hit the top of the keyboard.